src/backend/embedding/extract_file/extract_file.py

```diff
@@ -134,6 +137,10 @@ def extract_file(self, customer_guid: str, filename: str):
                 output_file_path = self.file_extract.extract_json_content(customer_guid, local_path, filename)
                 self.upload_extracted_content(customer_guid, filename, output_file_path)
                 return {"message": "JSON extracted and uploaded successfully."}
+            elif file_type == FileType.IMAGE:
+                output_file_path = self.file_extract.extract_image_content(customer_guid, local_path, filename)
+                self.upload_extracted_content(customer_guid, filename, output_file_path)
```

Comment:
Isnt these two lines same for all if condition?
Can we remove it from all if condition and move it below all if condition?

 self.upload_extracted_content(customer_guid, filename, output_file_path)
 return {"message": "JPEG or PNG or JPG extracted and uploaded successfully"}